Readme Azure Spring Cloud.md

```diff
@@ -0,0 +1,152 @@
+
+```bash
+    az extension add --name spring-cloud
+```
+Note - `spring-cloud` CLI extension `2.1.0` or later is a pre-requisite to enable the
+latest Java in-process agent for Application Insights. If you already 
+have the CLI extension, you may need to upgrade to the latest using --
+
+```bash
+    az extension update --name spring-cloud
+```
+### Provision Azure Spring Cloud service instance using Azure CLI
+
+### Prepare your environment for deployments
+Set the following environment variables required in the Azure Spring Cloud deployment below.
+
+```bash
+
+    export SUBSCRIPTION=subscription-id # customize this
+    ...
+    export RESOURCE_GROUP=resource-group-name # customize this
+    ...
+    export REGION=westus2 # customize this
+    ...
+    export SPRING_CLOUD_SERVICE=azure-spring-cloud-name # customize this
+    ...
+```
+### Login to Azure 
+Login to the Azure CLI and choose your active subscription. Be sure to choose the active subscription that is whitelisted for Azure Spring Cloud
+
+```bash
+    az login
+    az account list -o table
+    az account set --subscription ${SUBSCRIPTION}
+```
+
+### Create Azure Spring Cloud service instance
+Prepare a name for your Azure Spring Cloud service.  The name must be between 4 and 32 characters long and can contain only lowercase letters, numbers, and hyphens.  The first character of the service name must be a letter and the last character must be either a letter or a number.
+
+Create a resource group to contain your Azure Spring Cloud service.
+
+```bash
+    az group create --name ${RESOURCE_GROUP} \
+        --location ${REGION}
+```
+
+### Create an instance of Azure Spring Cloud.
+
+```bash
+    az spring-cloud create --name ${SPRING_CLOUD_SERVICE} \
+        --sku standard --enable-java-agent \
+        --resource-group ${RESOURCE_GROUP} \
+        --location ${REGION}
+```
+
+The service instance will take around five minutes to deploy.
+
+Set your default resource group name and cluster name using the following commands:
+
+```bash
+    az configure --defaults \
+        group=${RESOURCE_GROUP} \
+        location=${REGION} \
+        spring-cloud=${SPRING_CLOUD_SERVICE}
+```
+
+### Create microservice applications
+
+Create 5 microservice apps.
+
+```bash
+    az spring-cloud app create --name spring-petclinic-rest --instance-count 1 --assign-endpoint true \
+        --memory 2 \
+        --jvm-options='-Xms2048m -Xmx2048m'
+```
+
+
+### Deploy applications and set environment variables
+
+Deploy Spring Petclinic rest application to Azure.
+
+```bash
+    az spring-cloud app deploy --name spring-petclinic-rest \
+        --jar-path target/spring-petclinic-rest-2.4.2.jar \
+        --jvm-options='-Xms2048m -Xmx2048m'
+```
+
+## Swagger REST API documentation presented here (after application start):
+Copy the assinged url from the portal for the spring petclinic application deployed above. 
+
+[http://<url>/petclinic/swagger-ui.html](http:<url>/petclinic/swagger-ui.html)
```

Comment:
This URL is not rendering correctly

Reply:
The user has to copy the assigned end point from the portal and append /petclinic/swagger-ui.html to actually hit the endpoint. Currently it will not render to anything. I have removed the hyperlink.